docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -210,10 +209,11 @@ The point in time that the graphs start will be your back-filling end time. Chec
 
 Backfilling Process
 -------------------
-backup
+Backup
 ^^^^^^
-If you have a long retention period you are using an external directory that holds the Prometheus data, back it up, in case
-If you have a long retention period, you are using an external directory that holds the Prometheus data back it up; if something goes wrong in the process, you can revert the process.
+Backup any external directory containing Prometheus data; if something goes wrong, you can revert the changes.
```

Comment:
It's not any, it the external data directory

Reply:
The modified sentence would be:

Backup the external directory containing Prometheus data; if something goes wrong, you can revert the changes.